docs/api.md

```diff
@@ -1,6 +1,10 @@
 ## Classes
 
 <dl>
+<dt><a href="#Application">Application</a></dt>
+<dd></dd>
+<dt><a href="#Application">Application</a> : <code>Object</code></dt>
+<dd></dd>
```

Comment:
This seems to be the source of the duplication. The thing that Application methods return is also an Application for jsdoc... Perhaps rename it ApplicationResponse?